### PR TITLE
[LORE] Radio Lore Update & Subspace Purge

### PR DESCRIPTION
**Pull request by PopeDaveThe3th**
## What Does This PR Do

Updates the descriptions of most radio-related objects in the game.

Removes the word subspace from most radio-related systems since it didn't mean anything. (subspace swap is unchanged because alliteration is cool.)

## Why It's Good For The Game
Radios needed a bit of lore sprucing.

Subspace was referenced basically nowhere in a player-facing manner and doesn't exist in lore.

## Testing

- spawned in
- observed items
- observed telecom core
- checked the circuit imprinter
- checked AI radio
- all good here boss

<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  
  <hr>

## Changelog

:cl:
tweak: Updated radio lore (removing subspace while i was at it)
/:cl:

### CODE REVIEW

File: code/game/objects/items/devices/radio/headset.dm [hunk 1]
```diff
@@ -139,185 +139,185 @@
...
-	desc = "The headset of the man who protects your worthless lives."
+	desc = "An intimidating radio headset used by senior Asset Proection personnel."
 	icon_state = "com_headset"
 	item_state = "headset"
 	ks2type = /obj/item/encryptionkey/heads/hos
 
 /obj/item/radio/headset/heads/hos/alt
 	name = "head of security's bowman headset"
-	desc = "The headset of the man in charge of keeping order and protecting the station. Protects ears from flashbangs."
+	desc = "An intimidating tactical headset used by senior Asset Protection personnel."
```
Comment (Fordoxia) [inline code/game/objects/items/devices/radio/headset.dm:247]:
```suggestion
	desc = "An intimidating tactical headset used by senior Asset Protection personnel. Protects against loud noises."
```

File: code/game/objects/items/devices/radio/headset.dm [hunk 2]
```diff
@@ -139,185 +139,185 @@
...
-	desc = "The headset of the Magistrate."
+	desc = "A well-polished radio headset used by senior Nanotrasen legal specialists."
 	icon_state = "com_headset"
 	item_state = "headset"
 	ks2type = /obj/item/encryptionkey/heads/magistrate
 
 /obj/item/radio/headset/heads/magistrate/alt
 	name = "magistrate's bowman headset"
-	desc = "The headset of the Magistrate. Protects ears from flashbangs."
+	desc = "A tactical headset used by Nanotrasen magistrates."
```
Comment (Fordoxia) [inline code/game/objects/items/devices/radio/headset.dm:320]:
```suggestion
	desc = "A tactical headset used by Nanotrasen magistrates. Protects against loud noises."
```

File: code/game/objects/items/devices/radio/headset.dm [hunk 3]
```diff
@@ -361,7 +361,7 @@
 
 /obj/item/radio/headset/ert/alt/commander
 	name = "ERT commander's bowman headset"
-	desc = "The headset of the boss. Protects ears from flashbangs. Can transmit even if telecomms are down."
+	desc = "An advanced tactical headset used by high-ranking Nanotrasen PMCs. Equipped with an advanced transmission array and ear protection."
```
Comment (Fordoxia) [inline code/game/objects/items/devices/radio/headset.dm:364]:
```suggestion
	desc = "An advanced tactical headset used by high-ranking Nanotrasen PMCs. Equipped with an advanced transmission array that doesn't require a telecom core, and protects against loud noises."
```

**Review by Fordoxia — CHANGES_REQUESTED** [review]
Probably also add a description to the phone, as it exists in the world and lacks one - and will inherit the station bounced radio description otherwise.

**Comment by PopeDaveThe3th** [comment]
Laziness hit hard with this one. Probably going to remake it in the future but can't be bothered right now.

**Pull request closed without merging**